tests/terraform/checks/resource/aws/example_APIGatewayDeploymentCreateBeforeDestroy/main.tf

```diff
@@ -0,0 +1,33 @@
+resource "aws_api_gateway_deployment" "pass" {
+  rest_api_id = "some rest api id"
+  stage_name  = "some name"
+  lifecycle {
+    create_before_destroy=true
+  }
+  tags {
+    project = "ProjectName"
+  }
+}
+
+
+resource "aws_api_gateway_deployment" "fail" {
+  rest_api_id = "some rest api id"
+  stage_name  = "some name"
+  lifecycle {
+    create_before_destroy=false
+  }
+  tags {
+    project = "ProjectName"
+  }
+}
+
+resource "aws_api_gateway_deployment" "fail2" {
+  rest_api_id = "some rest api id"
+  stage_name  = "some name"
+  lifecycle {
+    create_before_destroy=false
+  }
+  tags {
+    project = "ProjectName"
+  }
+}
```

Comment:
one of those two should be without the lifecycle block 😄

Reply:
yeah must have missed the edit